docker/image/app/root/lib/task/modules/after.sh.twig

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
 
+{% set version = @('go.version') > 1.15 ? '@latest' : '' %}
```

Comment:
```suggestion
{% set version = version_compare(@('go.version'), '1.15', '>') ? '@latest' : '' %}
```
more flexible if using a specific bugfix release but requires https://github.com/inviqa/harness-base-php/blob/1.1.x/src/_base/harness/config/functions.yml#L211-L217

Reply:
otherwise there's also GOLANG_VERSION env var in the container

Reply:
👍 noted, thanks - I think comparing it just as a float is fine for now (works for 1.15 + 1.16 so should be fine)